pkg/apiserver/registry/stats/nodelatencystats/rest_test.go

```diff
@@ -13,3 +13,236 @@
 // limitations under the License.
 
 package nodelatencystats
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"k8s.io/apimachinery/pkg/apis/meta/internalversion"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+
+	statsv1alpha1 "antrea.io/antrea/pkg/apis/stats/v1alpha1"
+)
+
+func TestREST(t *testing.T) {
+	r := NewREST()
+	assert.Equal(t, &statsv1alpha1.NodeLatencyStats{}, r.New())
+	assert.Equal(t, &statsv1alpha1.NodeLatencyStats{}, r.NewList())
+	assert.False(t, r.NamespaceScoped())
+}
+
+func TestRESTCreate(t *testing.T) {
+	// Define the test case
+	name := "create summary"
+	summary := &statsv1alpha1.NodeLatencyStats{
+		ObjectMeta:           metav1.ObjectMeta{Name: "node1"},
+		PeerNodeLatencyStats: nil,
+	}
+	expectedObj := &statsv1alpha1.NodeLatencyStats{
+		ObjectMeta:           metav1.ObjectMeta{Name: "node1"},
+		PeerNodeLatencyStats: nil,
+	}
+	expectedErr := false
+
+	// Execute the test case
+	t.Run(name, func(t *testing.T) {
+		r := NewREST()
+		obj, err := r.Create(context.TODO(), summary, nil, nil)
+		if expectedErr {
+			assert.NotNil(t, err)
+		} else {
+			assert.Nil(t, err)
+			assert.Equal(t, expectedObj, obj)
+		}
+	})
+}
+
+func TestRESTGet(t *testing.T) {
+	tests := []struct {
+		name        string
+		summary     *statsv1alpha1.NodeLatencyStats
+		nodeName    string
+		expectedObj runtime.Object
+		expectedErr bool
+	}{
+		{
+			name: "get summary",
+			summary: &statsv1alpha1.NodeLatencyStats{
+				ObjectMeta:           metav1.ObjectMeta{Name: "node1"},
+				PeerNodeLatencyStats: nil,
+			},
+			nodeName: "node1",
+			expectedObj: &statsv1alpha1.NodeLatencyStats{
+				ObjectMeta:           metav1.ObjectMeta{Name: "node1"},
+				PeerNodeLatencyStats: nil,
+			},
+			expectedErr: false,
+		},
+		{
+			name: "get summary not found",
+			summary: &statsv1alpha1.NodeLatencyStats{
+				ObjectMeta:           metav1.ObjectMeta{Name: "node1"},
+				PeerNodeLatencyStats: nil,
+			},
+			nodeName:    "node2",
+			expectedObj: nil,
+			expectedErr: true,
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			r := NewREST()
+			_, err := r.Create(context.TODO(), tt.summary, nil, nil)
+			assert.Nil(t, err)
+			obj, err := r.Get(context.TODO(), tt.nodeName, nil)
+			if tt.expectedErr {
+				assert.NotNil(t, err)
```

Comment:
Ok.

Reply:
Use `assert.ErrorIs` may cause error not match, because the  Details pointer in error.StatusError is not match.
I've updated it to compare Error Code.

Reply:
In this case, maybe you can use `assert.EqualError`

Reply:
Yes, this function helps a lot.